mesonbuild/modules/pkgconfig.py

```diff
@@ -591,9 +602,11 @@ def parse_variable_list(vardict):
                 pkgroot_name = os.path.join('{libdir}', 'pkgconfig')
         if not isinstance(pkgroot, str):
             raise mesonlib.MesonException('Install_dir must be a string.')
+        relocatable = state.environment.coredata.get_option(mesonlib.OptionKey('relocatable', module='pkgconfig'))
```

Comment:
There is `state.get_option('relocatable', module='pkgconfig')` shortcut.

Reply:
Latest commit I changed to the shortcut.